apisix/admin/plugins.lua

```diff
@@ -37,7 +37,42 @@ function _M.stream_check_schema(plugins_conf, schema_type)
 end
 
 
+local function get_plugins_all_attributes()
+    local plugins = core.config.local_conf().plugins
+    local all_attributes = core.table.new(#plugins, 0)
+    for _, name in ipairs(plugins) do
+        local plugin_name = "apisix.plugins." .. name
+        local ok, plugin = pcall(require, plugin_name)
```

Comment:
There is not need to reload all the plugins. You can directly get them like:
https://github.com/apache/apisix/blob/0802c95faf4c5615c3b9fbe56ef913be7af08287/apisix/control/v1.lua#L37-L45

Reply:
This method is not provided when you wrote this PR. But I think now we can make it simpler.

Reply:
Is this pr still necessary?  I see https://github.com/apache/apisix/pull/3092 already contains the functions provided by this pr.

Reply:
It is different thing. #3092 is in control API.

Reply:
update